nuxdie/blog
# Update dependency react-helmet to version 5.x

This Pull Request updates dependency react-helmet from version `4.0.0` to `5.0.3`

### Changelog

#### 5.0.3 / 2017-04-10

  * docs: changelog 5.0.3 release
  * fix: React.propTypes -&gt; PropTypes ([#265](https://github.com/nfl/react-helmet/issues/265))
    * Update React.propTypes -&gt; PropTypes
    * add yarn.lock
  * docs: readme now reflects renderStatic provides title that renders whole tag ([#261](https://github.com/nfl/react-helmet/issues/261))
    * helmet does not contain titleAttributes
    * avoid duplication of title tags

#### 5.0.2 / 2017-03-28

  * docs: changelog 5.0.2 release
  * fix: prevent clearing existing title ([#259](https://github.com/nfl/react-helmet/issues/259))
    * test: added test to exemplify bug
    * fix: prevent clearing existing title
    * test: check for empty title
    * Bump version
  * chore: update code coverage ([#254](https://github.com/nfl/react-helmet/issues/254))
    * chore: combine code coverage from multiple browsers
    * chore: add firefox to browser list
    * test: cover requestIdleCallback
    * test: avoid running Firefox in Travis.

#### 5.0.1 / 2017-03-24

  * 5.0.1 ([#257](https://github.com/nfl/react-helmet/issues/257))
  * fix([#255](https://github.com/nfl/react-helmet/issues/255)): handle falsy children values ([#256](https://github.com/nfl/react-helmet/issues/256))

#### 5.0.0 / 2017-03-21

  * chore: add commitizen + conventional-changelog ([#253](https://github.com/nfl/react-helmet/issues/253))
    * chore: add commitizen
    add support for commitizen + conventional-changelog
    * Merge branch &#x27;master&#x27; into chore/codecov
  * chore: add codecov ([#251](https://github.com/nfl/react-helmet/issues/251))
  * release: Helmet 5.0.0
  * Helmet 5.0.0: requestIdleCallback support ([#248](https://github.com/nfl/react-helmet/issues/248))
    * (feat) - new API to proved actual head tags as children to Helmet creating a more declarative API.  Fully backwards compatible.
    * test: Update tests to use declarative API
    * Converting api -&gt; title unit tests.
    * ESLint fix.
    * Duplicate prop
    * title attributes unit tests
    * Update tests to use declarative API
    * ESLint fixes.
    * ref -&gt; rel
    * html Attributes unit tests
    * fix lang warning
    * test: Update onChangeClientState
    * test: Update script tags
    * test: Update noscript tags
    * Update baseTag + metaTag
    * test: Update style tags
    * partial link tag unit tests
    * more link tags
    * test: Update link tags
    * fix: Don&#x27;t warn if no child props are provided.
    * (fix) title and titleAttribute unit tests
    * test: Fix html attributes tests
    * fix: Group array-type children into arrays.
    * fix: Bridge nested children into appropriate child props.
    * fix: Noscript test
    * (fix) server meta unit test - special character
    * (fix) convert html attributes to react keys when rewinding as a component
    * fix: Update describe name.
    * (fix) trailing comma
    * Bump to 5.0.0-beta
    * feat: Add {Helmet} export
    * feat: Add renderStatic alias
    * test: Update grammar
    * refactor: Organize new children to props code
    * 5.0.0-beta.2
    * chore: Consolidate &amp; organize Helmet constants.
    * feat: Support body attributes
    * fix: render body attributes on the server
    * chore: update README
    * feat: use requestIdleCallback to batch DOM updates
    * chore: move utilities into separate file
    * fix: correct scope for native requestIdleCallback
    * fix: add cancelIdleCallback polyfill
    * fix: clear pending callbacks to prevent multiple DOM writes
    * fix: export requestIdleCallback
    * fix: track attributes using attributeKeys
    * test: make tests asynchronous &amp; update grammar
    * chore: improve warnings / errors
    * chore: update dev dependencies
    * fix: throw on invalid string as child
    * chore: update config
    * chore: update karma.config.js
    * test: fallback if mappedState is undefined
    * fix: avoid unnecessary DOM writes
    * feat: opt out of string encoding
    * chore: update documentation
    * chore: add note on backward compatibility
    * test: check for invalid self-closing tags
  * feat: refactor Helmet API ([#246](https://github.com/nfl/react-helmet/issues/246))
    * (feat) - new API to proved actual head tags as children to Helmet creating a more declarative API.  Fully backwards compatible.
    * test: Update tests to use declarative API
    * Converting api -&gt; title unit tests.
    * ESLint fix.
    * Duplicate prop
    * title attributes unit tests
    * Update tests to use declarative API
    * ESLint fixes.
    * ref -&gt; rel
    * html Attributes unit tests
    * fix lang warning
    * test: Update onChangeClientState
    * test: Update script tags
    * test: Update noscript tags
    * Update baseTag + metaTag
    * test: Update style tags
    * partial link tag unit tests
    * more link tags
    * test: Update link tags
    * fix: Don&#x27;t warn if no child props are provided.
    * (fix) title and titleAttribute unit tests
    * test: Fix html attributes tests
    * fix: Group array-type children into arrays.
    * fix: Bridge nested children into appropriate child props.
    * fix: Noscript test
    * (fix) server meta unit test - special character
    * (fix) convert html attributes to react keys when rewinding as a component
    * fix: Update describe name.
    * (fix) trailing comma
    * Bump to 5.0.0-beta
    * feat: Add {Helmet} export
    * feat: Add renderStatic alias
    * test: Update grammar
    * refactor: Organize new children to props code
    * 5.0.0-beta.2
    * chore: Consolidate &amp; organize Helmet constants.
    * feat: Support body attributes
    * fix: render body attributes on the server
    * chore: update README
    * chore: fix typos
